liquibase-core/src/main/java/liquibase/changelog/visitor/UpdateVisitor.java

```diff
@@ -40,8 +43,12 @@ public Direction getDirection() {
     @Override
     public void visit(ChangeSet changeSet, DatabaseChangeLog databaseChangeLog, Database database,
                       Set<ChangeSetFilterResult> filterResults) throws LiquibaseException {
+        Executor executor = Scope.getCurrentScope().getSingleton(ExecutorService.class).getExecutor("jdbc", database);
+        if (! (executor instanceof LoggingExecutor)) {
+            Scope.getCurrentScope().getUI().sendMessage("Running Changeset: " + changeSet);
```

Comment:
Is this logs directly to stdout or it should respect logging levels? On what level it logs then? I have `logging.level.liquibase: WARN` (in Spring Boot's `application.properties`) and after update I see all these messages in console as-is, without date/level/class/etc. For example:
```
2022-01-22 15:02:34.380 [user:  ]  INFO 18949 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
Running Changeset: liquibase/initial-state.xml::create-users_activation-table::php-coder
Running Changeset: liquibase/initial-state.xml::create-users_table::php-coder
Running Changeset: liquibase/initial-state.xml::create-countries-table::php-coder
...
```

Reply:
It could be related to my dated Spring Boot version, so I'm not fully sure whether it's a bug in Liquibase..

Reply:
Additional observation: these messages appears also when I run unit tests that use H2 :-/

Reply:
Agree on this one. We have a few hundreds of change logs and thousands of tests. Imagine how messy build logs are now.

Reply:
https://github.com/liquibase/liquibase/issues/2396